### PR TITLE
Fix RTL Customize Toggles & Dashboard

### PR DESCRIPTION
**Pull request by Qasem-h**
admin-cous
admin-couss

### CODE REVIEW

File: app/assets/stylesheets/common/base/rtl.scss [hunk 1]
```diff
@@ -38,6 +38,14 @@
 .rtl .pull-left {
   float:right !important;
 }
-.rtl .autocomplete {
-left: 360px  !important;
+
+left: 0px !important;
```
Comment (ZogStriP) [inline app/assets/stylesheets/common/base/rtl.scss]: I don't think you want that without a selector

Comment (Qasem-h) [inline app/assets/stylesheets/common/base/rtl.scss]: Sorry I forget to change it

**Comment by SamSaffron** [comment]
are we sure about this change to 

 .rtl .autocomplete  

sounds VERY wide to me and not specific to this page.

**Comment by Qasem-h** [comment]
Im sure about that,

**Comment by Qasem-h** [comment]
@ZogStriP  can you review this PR ?

**Comment by Qasem-h** [comment]
dashbord 
befro 
new0

after
new1

**Comment by Qasem-h** [comment]
and aotucomplite  : 
if 0px
new-p-0
and its now like this
new1

**Comment by SamSaffron** [comment]
@scossar can you review?

**Comment by scossar** [comment]
I'll look at it later today.

**Comment by scossar** [comment]
It's good. I think there are a couple of things need to be fixed.

Adding the `!important` flag to a direction makes it so that it is NOT flipped. That what we want here.
```
.rtl .customize .current-style .toggle-mobile {
  position: static !important;
  float: left !important; // This needs the !important flag
}

.rtl .customize .current-style .toggle-maximize {
  position: static !important;
  float: left !important; // This needs the !important flag
}
```

For `.dashboard-left` and `.dashboard-right` it might be good enough to do it like this. The width has already been defined and can be left out here. The `margin-top` is not in the main Discourse styles. Maybe there is a reason to use it here.
```
.rtl .dashboard-left {
  float: right !important;
}
.rtl .dashboard-right {
  float: left !important;
}
```

 `.rtl .autocomplete` doesn't need the `!important` flag. I think `.rtl .autocomplete { left: 27px; }` (which is flipped to `right: 27px;`) would be closer to the way it works in the left-to-right version of Discourse. I don't know if `.autocomplete` is used outside of the composer. If it is, this will have some kind of effect on it.

**Comment by SamSaffron** [comment]
@Qasem-h can you take care of these changes?

**Comment by Qasem-h** [comment]
@SamSaffron  Im tested it in localhost its works
and i well check it again TONIGHT

**Comment by Qasem-h** [comment]
I think .autocomplete left need just 1px or 0 px but im changed to 27 px

**Comment by scossar** [comment]
You should make it the way that looks correct to you.

**Comment by SamSaffron** [comment]
there are too many commits here, can you rebase as single commit and I will merge.

**Pull request closed without merging**